app/models/usage.rb

```diff
@@ -0,0 +1,6 @@
+class Usage < ApplicationRecord
+  belongs_to :subscription
+  belongs_to :feature
+
+  validates :feature_count, presence: true, numericality: { greater_than_or_equal_to: 0 }
```

Comment:
what would be the functionality if feature count is 0?

Reply:
It will simply charge Plan monthly fees.